### ISSUE TITLE
Provide data for order 243

### ISSUE THREAD

**Issue by olexandr-konovalov** [gap-packages/unitlib#10]
First, these need to be recalculated in the same way as #7. Second, the ability to retrieve them remotely should be fixed.

**Comment by olexandr-konovalov** [comment]
The new curlInterface package should provide a way to fix this.

**Comment by fingolfin** [comment]
<s>I think the `curlInterface` comment here was maybe intended for issue #3?</s> no it wasn't, but still, this second point is also in issue #3 :-)

**Comment by fingolfin** [comment]
By the way, I would recommend to put that data into this repository anyway; you can then add a `.release` script which removes it if for the release tarball. But this way, you have a guaranteed backup of it. Also, when downloading the files, that could be done via the GitHub server instead of a university server.

**Comment by olexandr-konovalov** [comment]
@fingolfin yes, 32 MB unpacked in `data/243`, 18 MB tar,.gz or 16 MB tar.bz - which is better?

**Comment by fingolfin** [comment]
git doesn't handle binaries very well, so I'd always put the unpacked data into the repo; .git compresses all blobs anyway, but this way it can e.g. handle diffs in the data should you ever have to tweak it.

**Comment by limakzi** [comment]
>
> git doesn't handle binaries very well, so I'd always put the unpacked data into the repo; .git compresses all blobs anyway, but this way it can e.g. handle diffs in the data should you ever have to tweak it.
>
 
@fingolfin The end-user will get uncompressed data. Is it OK?

**Comment by olexandr-konovalov** [comment]
@limakzi ok, I've reread this thread from the start, and the data I have need to be recalculated (of course, I can still put them in a branch now and later use that to show if the recalculation would result in any diffs, but it surely would). 

The thing is that in 2017 a broken example in the master branch (#6) resulted in a 2018 fix in LAGUNA (https://github.com/gap-packages/laguna/pull/6) and subsequent recalculation of UnitLib data in #7, which did not cover order 243 (since the latter stopped to be available in 2017, see #3).

Recalculating can be done using sequentially with https://github.com/gap-packages/unitlib/blob/master/lib/genlib.g, or in parallel using https://github.com/gap-packages/unitlib/blob/master/lib/parunits.g and the GAP SCSCP package (https://gap-packages.github.io/scscp/, see also https://github.com/olexandr-konovalov/scscp-demo).

Some numbers given in the 2010 paper "Parallel computations in modular group algebras" by @stevelinton and myself (https://doi.org/10.1145/1837210.1837231). For example, for the `SmallGroup(243,26)` which has structure description `"(C9 x C9) : C3"` sequential time was ~5 hours, parallelised version took just below an hour on 8-9 cores, 27 minutes on 17 cores, and 20 minutes on 25 cores. So, this sounds feasible.
